src/main/java/edu/uw/edm/edmzuulproxy/properties/CertificateAuthorizationProperties.java

```diff
@@ -21,4 +23,6 @@
 
     @NonNull
     private String certificateNameHeader;
+
+    private Map<String, String> certificateToApiKeyMap;
```

Comment:
Consider adding dev comments.

Reply:
I considered  adding a comment initially. But I thought the name, certificateToApiKeyMap, is fairly clear and additional comment would not provide much additional value.